library/src/androidTest/java/org/xmtp/android/library/GroupTest.kt

```diff
@@ -1028,7 +1028,7 @@ class GroupTest {
 
         assertEquals(runBlocking { alixGroup.messages() }.size, 1)
         assertEquals(runBlocking { alixGroup2.messages() }.size, 1)
-        assertEquals(numGroups, 3u)
+        assertEquals(numGroups, 2u) // I still think this should be 3
```

Comment:
@codabrink this makes me think it should be 3 still if it was syncing the sync group correctly?

Reply:
Apparently this is expected now that syncAll does not sync the sync Group.